### ISSUE TITLE
QSW-M2116P-2T2S not compatible

### ISSUE THREAD

**Issue by Noltari** [Noltari/aioqsw#2]
According to @Herbs851, [QSW-M2116P-2T2S isn't compatible with this library](https://github.com/home-assistant/core/issues/71394).
If anyone with the knowledge required to reverse engineer the API calls with Wireshark or that could capture the network traffic with the QNAP switch while logging in to the switch website and send it to me is willing to help we could try to add support for this device.

**Comment by Noltari** [comment]
Attaching dumps provided by Home Assistant users.
[qnap-qsw-m2116p-2t2s-wireshark.zip](https://github.com/Noltari/aioqsw/files/10078621/qnap-qsw-m2116p-2t2s-wireshark.zip)
[qnap-qsw-m2116p-2t2s-js.zip](https://github.com/Noltari/aioqsw/files/10078623/qnap-qsw-m2116p-2t2s-js.zip)

**Comment by abraxxa** [comment]
I own such a switch since yesterday and am willing to help.

**Comment by gomble** [comment]
hey @Noltari maybe this helps?:
https://github.com/mannkind/qnap-qsw

**Comment by Noltari** [comment]
> hey @Noltari maybe this helps?: https://github.com/mannkind/qnap-qsw

Yes, but sadly I don't have time to implement it, I can't test it because I haven't got that device and honestly it would be wasting my time for something I've got no real intereset in doing.
However, if anyone wants to add support for those devices I'd be open to review PRs for it.

**Comment by Orunitia-lab** [comment]
I know this hasn't updated in a while but I would like to add two additional Qnap with new API to the list, if there will be anyone assisting Noltari with this issue.
QSW-M3216R-8S8T and QSW-M2106PR-2S2T

**Comment by llo871** [comment]
I also own a QSW-M3216R-8S8T, and would love to see some support, as I am quite happy with the switch performance. I am also available for any testing.
